src/devices.js

```diff
@@ -171,16 +171,16 @@ var handleBootstrapError = (err, errM, bootstrapEvent, backToFunction) => {
       bootstrapEvent.emit("error", errM)
 }
 
-var instructBootstrap = (fastbootboot, images, bootstrapEvent) => {
+var instructBootstrap = (fastboot, images, bootstrapEvent) => {
```

Comment:
This should not be changed, actually

Reply:
I can revert this bit solely. Can you explain what fastbootboot is then?

Reply:
`fastboot boot` is a command to boot a specific kernel. See https://androidforums.com/threads/list-of-fastboot-command.714676/

the variable tells the function if the devices bootloader supports that command